doc/Overview/Mvux/Advanced/Pagination.md

```diff
@@ -0,0 +1,364 @@
+
+    <ListView ItemsSource="{Binding PeopleAuto}" ItemTemplate="{StaticResource PersonDataTemplate}" />
+</Page>
+```
+
+In addition, make sure the `DataContext` of the `Page` is set to the generated bindable model called `BindablePeopleModel` (*MainPage.xaml.cs*):
+
+```csharp
+public MainPage()
+{
+    this.InitializeComponent();
+
+    this.DataContext = new BindablePeopleModel(new PeopleService());
+}
+```
+
+> [!TIP]  
+> You can inspect the generated code by either placing the cursor on the word `BindablePeopleModel` and hitting <kbd>F12</kbd>, see other ways to inspect the generated code [here](xref:Overview.Mvux.Advanced.InspectGeneratedCode).
+
+As you can see, there's nothing special in the XAML code as MVUX is taking advantage of the tools already implemented with the `ListView`.
+
+- When the page load, the first 20 items are loaded (after a delay of one second - as simulated in the service).
+- If there's available space in the `ListView`, the next batch of 20 items will be requested from the Feed and loaded from the service thereafter.
+- When the user scrolls down and hits the bottom of the `ListView`, the next 20 items will be requested.
+- This behavior will follow as the user scrolls down and chases the service for more data until all items have been loaded.
+
+Here's what the app renders like:
+
+![A video of an app that implements automatic incremental loading](../Assets/PaginationIncrementalLoading.gif)
+
+> [!NOTE]  
+> The source-code for the sample app demonstrated in this section can be found [here](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MvuxHowTos/PaginationPeopleApp).
+
+## Offset pagination
+
+Offset pagination is controlled via a State that stores the current page index in the Model, and the List-Feed depending on it, using [the `Select` operator](xref:Overview.Mvux.Feeds#select-or-selectasync).  
+When the user requests a new page, the current page index state is updated, thereby updating the dependent collection List-Feed.
+
+Using the example started in [incremental loading above](#incremental-loading), we'll add another method to the service, which will disclose to the View how many items are there in total. Getting a count of items is more efficient than enumerating all entries. This is necessary to identify the total number of pages we have.
+
+### Service
+
+Here's the methods added to the service (*PeopleService.cs*):
+
+```csharp
+public IFeed<uint> PageCount =>
+    Feed.Async(async (ct) => await PeopleService.GetPageCount(DefaultPageSize, ct));
+```
+
+The signature of `GetPageCount` should also be added to the `IPeopleService` interface:
+
+```csharp
+ValueTask<uint> GetPageCount(uint pageSize, CancellationToken ct);
+```
+
+### Model
+
+Let's expand the Model with the following:
+
+```csharp
+public partial record PeopleModel(IPeopleService PeopleService)
+{
+    public IListFeed<Person> PeopleAuto ...// will not be used in this example
+
+    public IFeed<uint> PageCount =>
```

Comment:
Please rework this example to remove the GetPageCount on the service. The total number of items/pages should come as part of a call to fetch each page of data

Reply:
We have 3 examples in the pagination project. The middle example implements manual pagination. The page count is required in this example.